apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -2025,7 +2025,7 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 		return updater;
 	}
 
-	private getAnalyticsUpdateQueryBySentByAgent(
+	async getAnalyticsUpdateQueryBySentByAgent(
```

Comment:
```suggestion
	private getAnalyticsUpdateQueryBySentByAgent(
```

function is not doing anything async or i'm wrong?

Reply:
You're right :grimacing: Thanks for suggesting this change
However, we can't move this back to private since we're using this function outside of the class scope (so I just removed `async`)